sphinx_gallery/tests/reference_parse.txt

```diff
@@ -10,13 +10,15 @@
  ('code',
   '\ndef dummy():\n    """This should not be part of a \'text\' block\'"""\n\n    ######################################\n    # Comment inside code to remain here\n    pass\n\n# this should not be part of a \'text\' block\n\n'),
  ('text',
-  '\n####################################################################\n\nMaking a line cut in sphinx\n'),
```

Comment:
I don't see any consequence for now. Still, as a cautionary meassure I just changed the commit to make sure that all blocks end with at least one newline so that we don't have the this line break giving unwanted behavior.